lib/org-ruby/output_buffer.rb

```diff
@@ -88,6 +88,8 @@ def insert(line)
       when ([:definition_term, :list_item, :table_row, :table_header,
              :horizontal_rule].include? line.paragraph_type)
         @buffer << "\n" << line.output_text.strip
+      when line.paragraph_type == :table_separator
+        @buffer << "\n" << line.output_text.strip.tr('+', '|')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.